### DISCUSSION TITLE
Wrap the `/packages` page in an Electron app and use it as desktop GUI

### DISCUSSION THREAD

**Discussion by wizard-28** [pacstall/website#12] [Ideas]
Alternative: [Neutralino](https://neutralino.js.org/) instead of [Electron](https://www.electronjs.org/). It works in the same way but uses 95% less space because it doesn't ship an entire chromium browser (uses the system renderer).

Discussion by @saenai255

**Comment by wizard-28** [comment]
Should this app have it's own separate repository?

**Comment by Elsie19** [comment]
We should totally go with Neutralino, *if* we decide to make an app at all

**Reply by wizard-28** [reply]
What about a GNOME store extension. IIRC someone requested that.

**Reply by Elsie19** [reply]
Yes, Renny on Discord mentioned that to me a couple times in DM's

**Comment by Zahrun** [comment]
I’d like to see a plasma-discover-backend-pacstall as there is already a plasma-discover-backend-flatpak and plasma-discover-backend-snap, etc.
I thought of starting the project, but sadly don’t have time for it.
